src/components/StoriesList/StoriesList.module.scss

```diff
@@ -1,82 +1,130 @@
 @import "../../styles/variables";
 
 .stories {
-  background: $grey;
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding-bottom: 50px;
+  padding-bottom: 30px;
+  margin-top: 50px;
+  background: lighten($grey, 10%);
 }
 
 .stories_list {
   display: flex;
+  flex-flow: row wrap;
   justify-content: center;
   max-width: 1000px;
   margin: 0 auto;
 
   a {
-    text-decoration: none;
     display: flex;
     flex-direction: column;
     justify-content: space-between;
     align-items: center;
+    text-decoration: none;
     background: $blue;
-    border-radius: 36px;
-    width: 400px;
+    border-radius: 30px;
+    max-width: 400px;
     height: 330px;
     margin: 20px;
     padding: 20px;
-    transition: 0.3s;
+    box-shadow: 6px 6px $dark-blue;
 
     &:hover {
       background-color: lighten(adjust-hue($blue, -3), 4%);
-      background-size: 200%;
+
+      .dial:nth-of-type(1) {
+        transform: rotate(180deg);
+      }
+
+      .led {
+        background-color: $yellow;
+      }
+
+      .screen {
+        background-size: 175%;
```

Comment:
Not clear on how this is working but you did it. I _do_ think the effect is too drastic. I think if it only moved 1/4 as much it would read a lot better. But I see that it's an aesthetic decision.

Reply:
Or maybe if the opacity was lowered so the filter is just generally more subtle?